packages/core/src/components/menu/_menu.scss

```diff
@@ -195,6 +197,9 @@ Markup:
 Styleguide components.menu.css
 */
 
+$menu-item-color-active: $minimal-button-background-color-active !default;
+$dark-menu-item-color-active: $dark-minimal-button-background-color-active !default;
```

Comment:
why are these defined here but `$menu-item-color-hover` is defined in _common.scss? i think all vars should be in _common.